src/Service/EnforcementManager.php

```diff
@@ -46,44 +49,50 @@ class EnforcementManager extends SS_Object
     private static $enabled = true;
 
     /**
-     * Whether the current member can skip the multi-factor authentication registration process.
+     * Whether the provided member can skip the MFA registration process.
      *
      * This is determined by a combination of:
-     *  - Whether MFA is required or optional
-     *  - If MFA is required, whether there is a grace period
-     *  - If MFA is required and there is a grace period, whether we're currently within that timeframe
+     *
+     *  - Whether MFA is enabled and there are methods available for use
+     *  - Whether the user has admin access (MFA is disabled by default for users that don't)
+     *  - Whether MFA is required - @see EnforcementManager::isMFARequired()
+     *  - Whether the user has registered MFA methods already
      *
      * @param Member&MemberExtension $member
      * @return bool
      */
     public function canSkipMFA(Member $member): bool
     {
+        if (!$this->isEnabled()) {
+            return true;
+        }
+
+        if ($this->config()->get('requires_admin_access') && !$this->hasAdminAccess($member)) {
+            return true;
```

Comment:
It's a bit of a confusing situation - the point is that admin access is required to _use_ MFA by default, as there's no built-in way for a user to manage their MFA configuration outside the admin UI. Therefore, users that _don't_ have admin access can skip anything MFA-related (unless configured otherwise.)

Reply:
could you put in a comment explaining this